### PR TITLE
Add domu config for NFS no coproc

### PR DESCRIPTION
**Pull request by andr2000**
This configuration file is about to be used for
NFS root without coproc enabled.

Signed-off-by: Oleksandr Andrushchenko <oleksandr_andrushchenko@epam.com>

### CODE REVIEW

File: recipes-dom0/dom0-image-weston/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/domu-nfs-no-coproc.cfg [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+seclabel='system_u:system_r:domU_t'
+
+# Guest name
+name = "DomU"
+
+# Kernel image to boot
+kernel = "/boot/domu/Image"
+
+device_tree = "/xen/domu.dtb"
```
Comment (aanisov) [inline recipes-dom0/dom0-image-weston/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/domu-nfs-no-coproc.cfg]: Why do you still need this DT? You claimed that no coprocs are incorporated, and there is no the rest of passthrough stuff.

Comment (otyshchenko1) [inline recipes-dom0/dom0-image-weston/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/domu-nfs-no-coproc.cfg]: AFAIK he uses DT for passing "pvrkm" node to the guest taking advantage of the hole in libxl (all stuff under passthrough directory are being copied as is).

Comment (andr2000) [inline recipes-dom0/dom0-image-weston/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/domu-nfs-no-coproc.cfg]: yes, this was my intention. but strictly speaking this has no relation to "no-coproc", so @aanisov comment is valid

Comment (aanisov) [inline recipes-dom0/dom0-image-weston/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/domu-nfs-no-coproc.cfg]: Passing the pvr to guest will include correspondent irq, mmio, etc. It is not enough to get something described in dtb for domu. As I understand all that stuff would be included in a separate patch, so this device_tree line should be the part of next patch.
